migtests/tests/analyze-schema/dummy-export-dir/schema/tables/table.sql

```diff
@@ -384,4 +384,82 @@ CREATE TABLE public.locations (
     created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
 );
 
-CREATE TABLE image (title text, raster lo);
\ No newline at end of file
+CREATE TABLE image (title text, raster lo);
+
+-- create table with multirange data types
+
+CREATE TABLE bigint_multirange_table (
+    id integer NOT NULL,
+    value_ranges int8multirange
+);
+
+
+CREATE TABLE date_multirange_table (
+    id integer NOT NULL,
+    project_dates datemultirange
+);
+
+
+CREATE TABLE int_multirange_table (
+    id integer NOT NULL,
+    value_ranges int4multirange
+);
+
+CREATE TABLE numeric_multirange_table (
+    id integer NOT NULL,
+    price_ranges nummultirange
+);
+
+
+CREATE TABLE timestamp_multirange_table (
+    id integer NOT NULL,
+    event_times tsmultirange
+);
+
+
+CREATE TABLE timestamptz_multirange_table (
+    id integer NOT NULL,
+    global_event_times tstzmultirange
+);
+
+
+ALTER TABLE ONLY bigint_multirange_table ALTER COLUMN id SET DEFAULT nextval('multirange_demo.bigint_multirange_table_id_seq'::regclass);
+
```

Comment:
same comment as above.

Reply:
actually you can even get rid of it, looks like we are not making use of these sequence/defaultvals

Reply:
Yeah removed the sequence/defaultvals and merged Primary key into the create tables itself.